### ISSUE TITLE
Choose whitelist folders

### ISSUE THREAD

**Issue by russianspy1234** [VinylMusicPlayer/VinylMusicPlayer#872]
**Is your feature request related to a problem? Please describe.**
I get random recordings and stuff in my library. I keep having to blacklist more and more folders. I keep my music on my SD card, but there does not seem to be an option to choose a different Folder than the internal storage one it defaults to

**Describe the solution you'd like**
Add a picker to the whitelist like the one for black list

**Describe alternatives you've considered**
Blacklisting a dozen folders

**Comment by AltayYuzeir** [comment]
Hello, thank you for this awesome FOSS project. I am enjoying it a lot! And love its modern look and feel and all the customization options. In the current setup of the app I can choose a root folder, where all my music is located, the Music folder in storage, and its subfolders. However, I was unable to add media subfolders which are in the Download folder for example. Can I add extra folders (i.e. folders not in the Root location) to the "Folders" tab in the side menu?
Thank you once again.

**Comment by soncaokim** [comment]
The current workflow is to have all musics under a single base folder. 
Starting with Android 13, things become messy: that base folder must not be the root folder, nor the Download folder itself (a subfolder of Download is okay), probably further restrictions will come too.

Do you mean that you want to have multiple base folders?

**Comment by AltayYuzeir** [comment]
Yeah maybe, something like that. Or be able to mark different folders as favorites for example. I also noticed one can search for a song only in the library side menu tab and nowhere else. I feel like having a search functionality in the folders tab of the side menu can lower the usability friction. In any case, I was looking for a FOSS tool to replace my old player, and VMP is the best product I encountered so far. Thank you.

**Comment by russianspy1234** [comment]
I've got a folder called music on my SD card that I would wanna use. Instead I have over 20 folders blacklisted

**Comment by soncaokim** [comment]
> I've got a folder called music on my SD card that I would wanna use. Instead I have over 20 folders blacklisted

Can you browse to the SD Card folder that contains your music?
<img width="336" alt="Screenshot 2024-03-28 at 09 20 57" src="https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/13333482/01c0aa79-58c9-4045-b90a-4cd6876e6e9f">

If yes, go to the folder containing your music, and set it as the start folder
<img width="331" alt="Screenshot 2024-03-28 at 09 21 27" src="https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/13333482/c38a0ffb-787d-4f48-b700-b3785498b75e">

Once done, you can go to the Settings page and white list only that start folder (ie. music from other folder wont be shown in the Library page)
<img width="335" alt="Screenshot 2024-03-28 at 09 21 53" src="https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/13333482/35c4ad4d-fa78-4db2-b063-f14c31a404ee">

**Comment by soncaokim** [comment]
> I feel like having a search functionality in the folders tab of the side menu can lower the usability friction.

You do have the Search button in the Library view here.
Otherwise I dont get your point :-)
<img width="317" alt="Screenshot 2024-03-28 at 09 29 55" src="https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/13333482/f726aa1b-564f-44ca-a880-5b72c43f656c">

**Comment by AltayYuzeir** [comment]
Please excuse me, perhaps I was unclear. I am attaching a screenshot.
![MVP](https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/34285813/ec27f57d-9d07-43b7-8aa3-a8debee4a73a)

**Comment by russianspy1234** [comment]
I can browse to it but I don't have option to set as start directory. Only options are scan directory and sort order

**Comment by soncaokim** [comment]
My understanding is that this issue is solved. Please reopen it if it's not the case (and help me to understand why).

**Comment by mockmeup** [comment]
@russianspy1234 
tldr; Please make sure you have the latest version installed (which currently is 1.11.0).

I had the same struggle and didn't have an "SD Card" entry in the menu (which opens on the left when you click to top left hamburger icon). In my case this was caused by an old version (1.4.0) which I couldn't update. Playstore wouldn't update it, so I deleted the app and installed it again. Now it's working!
